drivers/gpio/gpio_pca_series.c

```diff
@@ -0,0 +1,1727 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file Driver for PCA(L)xxxx SERIES I2C-based GPIO expander.
+ */
+
+#include <errno.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/init.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/gpio/gpio_utils.h>
+#include <zephyr/drivers/i2c.h>
+
+#define LOG_LEVEL CONFIG_GPIO_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(gpio_pca_series);
+
+/** Private debug macro, enable more error checking and print more log. */
+/* #define GPIO_NXP_PCA_SERIES_DEBUG */
+
+#define UNUSED(x)           (void)(x)
```

Comment:
We can drop this and use `ARG_UNUSED`, right?

Reply:
dropped.